src/config/stylegan.py

```diff
@@ -0,0 +1,95 @@
+import torch
+from loguru import logger as info_logger
+from src.disk import disk
+from pathlib import Path
+from src.logger.simple import Logger
+from src.data.baseline import  BaselineDataset
+from src.utils.download import download_data, unarchieve 
+from src.models.stylegan import StyleBased_Generator
+from src.training.stylegan import StyleGanTrainer
+from src.storage.simple import Storage
+from src.losses.perceptual import VGGPerceptualLoss
+from src.losses.ocr import OCRLoss
+from torchvision import models
+from torchvision.models.resnet import BasicBlock
+from torch.utils.data import DataLoader
+
+class ContentResnet(models.ResNet):
+    def _forward_impl(self, x):
+        # See note [TorchScript super()]
+        x = self.conv1(x)
+        x = self.bn1(x)
+        x = self.relu(x)
+        x = self.maxpool(x)
+
+        x = self.layer1(x)
+        x = self.layer2(x)
+        x = self.layer3(x)
+        x = self.layer4(x)
+
+        #x = self.avgpool(x)
+        #x = torch.flatten(x, 1)
+        #x = self.fc(x)
```

Comment:
In a perfect world these should have a note why they were commented out or deleted altogether, but I'm not saying that I'm not nitpicking right now.

Reply:
I think it's alright how it is, because it's clear how did we modify this block.